packages/select/src/withCreateRemoveOnDeleteForward.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    isBlockAboveEmpty,
+    isSelectionExpanded,
+    PlateEditor,
+    removeNodes,
+    Value,
+    WithPlatePlugin,
+  } from '@udecode/plate-common';
+  
+  import { RemoveOnDeleteForwardPlugin } from './createRemoveOnDeleteForwardPlugin';
+  
+  /**
+   * Set a list of element types to select on backspace
+   */
+  export const withCreateRemoveOnDeleteForward = <
+    V extends Value = Value,
+    E extends PlateEditor<V> = PlateEditor<V>,
+  >(
+    editor: E,
+    {
+      options: { query, removeNodeIfEmpty },
+    }: WithPlatePlugin<RemoveOnDeleteForwardPlugin, V, E>
+  ) => {
+    const { deleteForward } = editor;
+  
+    editor.deleteForward = (unit: 'character' | 'word' | 'line' | 'block') => {
```

Comment:
Use type inference here

Reply:
I did not get the idea of this, can you please share some reference.

Reply:
You can remove `unit` type, it's already inferred.

Reply:
@archie9211 Sorry, I marked this as resolved without realising it hadn't been done yet. Can you make this change?